achievements/urls-frontend.py

```diff
@@ -6,6 +6,8 @@
     path('', views.viewAllAchievements, name='achievements-frontend'),
     path('<int:id>/', views.viewSpecificAchievement, name='achievement-frontend'),
 
-    path('categories', views.viewAllCategories, name='categories-frontend'),
+    path('categories/', views.viewAllCategories, name='categories-frontend'),
     path('categories/<int:id>/', views.viewSpecificCategory, name='category-frontend'),
+
+    path('members/<int:id>/', views.viewSpecificMember, name='achievement-member-frontend'),
```

Comment:
Doing this allows everyone to see the achievements of any member (as anyone can access this page). I'm not sure if we want this.

Reply:
This is indeed something we need to think about, but at the moment people are able to see all people that have gained an achievement via the view_all_achievement page already.